crates/vm/levm/src/gas_cost.rs

```diff
@@ -343,9 +336,6 @@ fn mem_expansion_behavior(
     static_cost: u64,
 ) -> Result<u64, VMError> {
     let memory_expansion_cost = memory::expansion_cost(new_memory_size, current_memory_size)?;
-    let memory_expansion_cost: u64 = memory_expansion_cost
-        .try_into()
-        .map_err(|_| VMError::RevertOpcode)?;
```

Comment:
Note to reviewers: This error was introduced in the first upgrade from U256 to u64: https://github.com/lambdaclass/ethrex/commit/2bb8fa9b455c8f0ca51e8a4e26af63e2b915ce4d (coincidentally, by me) here: https://github.com/lambdaclass/ethrex/pull/1528/files#diff-4cd3c613c7078910433b6c76836c29fb05ba2675e53958ce6cadbfb9cf1292e3R339-R342.

I can't figure out for the life of me why I introduced that error instead of "VeryLargeNumber". The same number of test pass, so everything _seems_ to be functioning. 

It's strange, normally I'd write some sort of comment when things like this are important; maybe I simply made a mistake and I'm overthinking it.